### ISSUE TITLE
non-default `bleopt complete_menu_style` throws error in rcfile

### ISSUE THREAD

**Issue by rux616** [akinomyoga/ble.sh#54]
**ble version**: 0.4.0-devel2+61f9e10
**Bash version**: 5.0.16(1)-release,x86_64-pc-linux-gnu

When loading the blesh rcfile, setting the ble.sh option `complete_menu_style` to anything that is not `align-nowrap` throws an error: 

```
bleopt: Invalid value complete_menu_style='align'. A function 'ble/complete/menu-style:align/construct' is not defined.
```

**Comment by akinomyoga** [comment]
Thank you again for the report! I have fixed this 8a9a386. Can you check?

Originally these completion settings were intended to be specified in [`complete_load` hook](https://github.com/akinomyoga/ble.sh/wiki/Manual-%C2%A77-Completion#7-completion) but later allowed to be directly specified in `blerc`. But there was an oversight of needed changes.

**Comment by rux616** [comment]
> Thank you again for the report! I have fixed this [8a9a386](https://github.com/akinomyoga/ble.sh/commit/8a9a386c9a22c405dbcce6c3f4f1b1079c6558d2). Can you check?

It's working now!

> Originally these completion settings were intended to be specified in [`complete_load` hook](https://github.com/akinomyoga/ble.sh/wiki/Manual-%C2%A77-Completion#7-completion) but later allowed to be directly specified in `blerc`. But there was an oversight of needed changes.

I feel dumb. It's spelled out literally right there at the top of that manual page. Well, thanks for implementing it. :)

**Comment by akinomyoga** [comment]
Thank you for checking!
